src/avitab/apps/MapApp.cpp

```diff
@@ -261,6 +274,111 @@ void MapApp::selectEPSG() {
     chooserContainer->setVisible(true);
 }
 
+void MapApp::selectOnlineMaps() {
+    auto showOnlineMapsError([this](std::vector<std::string> errorMsgs) {
+        // Lambda function to show an error in the online maps window
+        // when we have trouble loading the user-defined online map
+        // configuration. JSON configs are very delicate (an extra comma
+        // can cause trouble), so it will be annoying if the user configured
+        // some maps, but can't see anything when starting AviTab. At least
+        // an error message briefing the user and advising them to look at
+        // the AviTab logs for more information improves the user experience
+        containerWithClickableList = std::make_unique<ContainerWithClickableCustomList>(
+                &api(), "Error loading online maps");
+        containerWithClickableList->setListItems(errorMsgs);
+        containerWithClickableList->setSelectCallback([](int selectedItem) {});
+        containerWithClickableList->setCancelCallback([this] () {
+            api().executeLater([this] () {
+                containerWithClickableList.reset();
+                chooserContainer->setVisible(false);
+            });
+        });
+        containerWithClickableList->show(chooserContainer);
+        chooserContainer->setVisible(true);
+    });
+
+    slippyMaps.clear();
+    std::vector<std::string> slippyMapNames;
+
+    // Read the config
+    std::string mapConfigPath(api().getDataPath() + "online-maps/mapconfig.json");
+    std::ifstream mapConfigFstream(fs::u8path(mapConfigPath));

```

Comment:
`std::ifstream` doesn't support UTF-8 paths on Windows, it expects ANSI by default. Please see the [comment in Platform.h](https://github.com/fpw/avitab/blob/f1d1e30f06658c51d77fdbebadf35ad4863b52fd/src/platform/Platform.h#L35) and use `fs::ifstream` instead, it's a drop-in replacement that works correctly on all platforms.